pages/osx/security.md

```diff
@@ -13,4 +13,12 @@
 
 - Create a keychain:
 
-`security create-keychain -p {{password}} {{name.keychain}}`
+`security create-keychain -p {{password}} {{keychain.name}}`
+
+- Set a certificate to use with a website or [s]ervice by its [c]ommon name (this will fail if several certificates with the same common name exist):
```

Comment:
```suggestion
- Set a certificate to use with a website or service by its common name (fails if several certificates with the same common name exist):
```

Reply:
Please see my comment https://github.com/tldr-pages/tldr/pull/6645/files#r720561999 , I'm unsure based on other pages if this is part of the repository's convention

Reply:
Implemented the "this will fail" -> "fails" change